custom/up_nrhm/data_sources/asha_facilitators.json

```diff
@@ -0,0 +1,170 @@
+{
```

Comment:
@kkrampa we're wanting to use these new data source configurations to replace fluff pillows. There is documentation on them here: https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/userreports/README.md